matching-core/src/main/java/com/graphhopper/matching/util/ViterbiMatchEntry.java

```diff
@@ -67,7 +67,7 @@ public int compare(QueryResult o1, QueryResult o2) {
         }
     };
 
-    public TimeStep(MatchEntry matchEntry) {
+    public ViterbiMatchEntry(MatchEntry matchEntry) {
```

Comment:
`ViterbiMatchEntry` sounds like this class represents the output of the Viterbi matching but it is actually the input. What about `HmmTimeStep`?